transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
You can disable checks for a specific file as well, add
```
# pylint: disable=C0321
```
to the file. (Can be done locally as well, by only including it in a block)

I just read that starting from Pylint v. 0.25.3, you can use the symbolic names for messages instead of remembering all those code numbers. E.g.:

```
# pylint: disable=missing-docstring
```

Current version is 1.4, so that would make things a bit more readable.